training/enhanced_dataset.py

```
@@ -0,0 +1,664 @@
+                    audio_path = self.negative_files[audio_idx]
+
+                result = self._load_and_process_audio(audio_path, label)
+
+            # Log performance for slow operations
+            load_time = time.time() - start_time
+            if load_time > 0.1:  # Log slow operations
+                logger.debug(f"Slow item load: idx={idx}, time={load_time:.3f}s, source={source}")
+
+            return result
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)